schema/example.go

```diff
@@ -0,0 +1,173 @@
+package schema
+
+func NewExample() Schema {
+	location := locationSchema{
+		Lat:  35.712758,
+		Long: 51.392114,
+	}
+
+	score := scoreSchema{
+		Type:  "GPA",
+		Value: "3.4",
+	}
+
+	resource := resourceSchema{
+		URL:   "http://www.example.com/my-example-slides/",
+		Label: "Slides",
+	}
+
+	core := coreSchema{
+		Name:              "John Doe",
+		Title:             "Software Engineer",
+		Image:             "example.com/Abcxyz",
+		Email:             "lucas@example.com",
+		Phone:             "http://www.example.com/my-slides/",
+		URL:               "http://www.example.com/",
+		Summary:           "The man who sold the world!",
+		CurrentLocation:   location,
+		PermanentLocation: location,
+	}
+
+	work := workSchema{
+		Name:        "XYZ Inc",
+		Description: "A social media company",
+		Position:    "Software Engineer",
+		Location:    location,
+		URL:         "http:/xyz.example.com",
+		StartDate:   "2017-12-29",
+		EndDate:     "2018-12-29",
+		Summary:     "Developing and maintaining the company website using syna",
+		Highlights:  []string{"Worked with mobile team at Twitter to develop remote debugging tools for mobile browsers"},
+	}
+
+	education := educationSchema{
+		Institution: "e.g. XYZ Institute of Technology",
+		Location:    location,
+		Area:        "Engineering",
+		StudyType:   "Bachelor",
+		StartDate:   "2017-06-28",
+		EndDate:     "2013-06-28",
+		Score:       score,
+		Courses:     []string{"CS302 - Introduction to Algorithms "},
+		Honors:      []string{"Magna Cum Laude"},
+	}
+
+	volunteer := volunteerSchema{
+		Organization: "Xyz",
+		Position:     "Open Source Contributor",
+		Location:     location,
+		URL:          "http://xyz.example.com",
+		StartDate:    "2014-06-29",
+		EndDate:      "2017-06-29",
+		Summary:      "Frontend developer",
+		Highlights:   []string{"Invited as a speaker in Xyzcon'17"},
+	}
+
+	publication := publicationSchema{
+		Name:        "Deep learning and Artificial Intelligence",
+		Publisher:   "XYZ, Computer Magazine",
+		ReleaseDate: "2015-08-01",
+		Resources:   []resourceSchema{resource},
+		URL:         "http://www.computer.org.example.com/csdl/mags/co/2015/10/rx069-abs.html;format:uri",
+		Summary:     "Discussion of the advent of deep learning and artificial intelligence",
+	}
+
+	legal := legalSchema{
+		Name:            "XYZ's patent on LZW compression, a fundamental part of the widely used GIF graphics format",
+		LegalType:       "Patent, Trademark, Copyright",
+		Description:     "Some legal document!",
+		ApplicationDate: "2015-08-01",
+		GrantDate:       "2016-09-01",
+		EndDate:         "2020-09-3",
+		Resources:       []resourceSchema{resource},
+		IDNumber:        "JP2004369746A",
+	}
+
+	skill := skillSchema{
```

Comment:
Concerning your question around the correct schema idea:
It can be a general area or specific.
A non specific one could be name = `Web Development`
With multiple keywords  such as `Go, Python, Backend`
The score would be scoped to the skill not to a specific keyword.

The specific case would be name = `Go backend`
With a score and either none or keywords such as `Distributed, Low Latency"

Reply:
I think this way it makes more sense. Fixed in 570748d476f068d3be54b49ed54ae44398a2eeaa